static/app/views/settings/account/notifications/fields2.tsx

```diff
@@ -177,4 +177,13 @@ export const QUOTA_FIELDS = [
       ['never', t('Off')],
     ] as const,
   },
+  {
+    name: 'quotaSpendAllocations',
```

Comment:
@cathyteng17 wait is there a backend notification set up for it yet? I think in the past when we had this sort of situation we hid the new field behind a feature flag.

Reply:
i'm working on it https://github.com/getsentry/getsentry/pull/8552

Reply:
but yes it would probably be good to put behind a feature flag -- how do you do that?